packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/features/exposureoffset/ExposureOffsetFeature.java

```diff
@@ -53,41 +52,41 @@ public void updateBuilder(CaptureRequest.Builder requestBuilder) {
       return;
     }
 
-    requestBuilder.set(CaptureRequest.CONTROL_AE_EXPOSURE_COMPENSATION, (int) currentSetting.value);
+    requestBuilder.set(CaptureRequest.CONTROL_AE_EXPOSURE_COMPENSATION, (int) currentSetting);
   }
 
   /**
-   * Return the minimum exposure offset double value.
+   * Returns the minimum exposure offset value, in counts of {@link #getExposureOffsetStepSize}.
```

Comment:
The return value is multiplied by `stepSize`, so this seems wrong.

Reply:
I was strongly influenced by the [documentation](https://developer.android.com/reference/android/hardware/camera2/CameraCharacteristics.html#CONTROL_AE_COMPENSATION_RANGE) from the Android Camera2 API when I documented this method.

Here this mention:

> Maximum and minimum exposure compensation values for android.control.aeExposureCompensation, in counts of android.control.aeCompensationStep, that are supported by this camera device.

I am not saying this would be necessarily correct or the best, just wanted to quickly verify with you before I update it.

Reply:
I don't follow. You're pointing to the documentation for the exposure range, which is line 64 here. This method doesn't return that, it returns that multiplied by the step size (line 67). If you take step counts, and multiply by step sizes, you don't have step counts any more. You have the actual exposure value.

Reply:
Yes you are right, I completely read the Android documentation wrong.

Reply:
Fixed